scripts/globals/mobskills/burning_blade.lua

```diff
@@ -18,8 +18,8 @@ mobskillObject.onMobSkillCheck = function(target, mob, skill)
 end
 
 mobskillObject.onMobWeaponSkill = function(target, mob, skill)
-    local dmgmod = 1
-    local info = xi.mobskills.mobMagicalMove(mob, target, skill, mob:getMobWeaponDmg(xi.slot.MAIN), xi.magic.ele.FIRE, dmgmod, xi.mobskills.magicalTpBonus.DMG_BONUS, 1, 0, 1.5, 1.75, 2)
+    local dmgmod = 1.8
+    local info = xi.mobskills.mobMagicalMove(mob, target, skill, mob:getMobWeaponDmg(xi.slot.MAIN) * 3, xi.magic.ele.FIRE, dmgmod, xi.mobskills.magicalTpBonus.NO_EFFECT)
```

Comment:
As per our secret DMs. Take out the *3 and leave  `NO_EFFECT, 1, 0, 2, 2.75, 3)`
Set `dmgmod = 1`